Sources/PackageDescription/SupportedPlatforms.swift

```diff
@@ -161,6 +164,25 @@ public struct SupportedPlatform: Encodable, Equatable {
     public static func watchOS(_ versionString: String) -> SupportedPlatform {
         return SupportedPlatform(platform: .watchOS, version: SupportedPlatform.WatchOSVersion(string: versionString).version)
     }
+
+    /// Configures the minimum deployment target version for the DriverKit platform.
+    ///
+    /// - Parameter version: The minimum deployment target that the package supports.
+    @available(_PackageDescription, introduced: 999.0)
+    public static func driverKit(_ version: SupportedPlatform.DriverKitVersion) -> SupportedPlatform {
+        return SupportedPlatform(platform: .driverKit, version: version.version)
+    }
+
+    /// Configures the minimum deployment target version for the DriverKit platform
+    /// using a custom version string.
+    ///
+    /// The version string must be a series of two or three dot-separated integers, such as `19.0` or `19.0.1`.
+    ///
+    /// - Parameter versionString: The minimum deployment target as a string representation of two or three dot-separated integers, such as `2.0.1`.
```

Comment:
s/2.0.1/19.0.1/

Reply:
Ah lol, I changed one string but not the other 🤦